BaseFilter/sections/specific.txt

```diff
@@ -154,6 +154,13 @@ youtubekids.com,youtube-nocookie.com,youtube.com#%#//scriptlet('set-constant', '
 youtubekids.com,youtube-nocookie.com,youtube.com#%#//scriptlet('set-constant', 'ytInitialPlayerResponse.playerAds', 'undefined')
 youtubekids.com,youtube-nocookie.com,youtube.com#%#//scriptlet('set-constant', 'playerResponse.adPlacements', 'undefined')
 !
+! Remove anti-adb
+!#if (adguard_app_windows || adguard_app_mac || adguard_app_android || adguard_ext_firefox)
+.com/watch?v=$replace=/"auxiliaryUi.*?e\}\}\}\,//,domain=www.youtube.com
+.com/watch?v=$replace=/("trackingParam":"kx_fmPxhoPZR)[-_0-9A-Za-z]{275}([-_0-9A-Za-z]{34}zRgkuMsNLBwOcCE59TDtslLKPQ-SS"\})/\$1\$2/,domain=www.youtube.com
+/youtubei/v*/player?key=$replace=/"auxiliaryUi.*?e\}\}\}\,//,domain=www.youtube.com
+/youtubei/v*/player?key=$replace=/("trackingParam":"kx_fmPxhoPZR)[-_0-9A-Za-z]{275}([-_0-9A-Za-z]{34}zRgkuMsNLBwOcCE59TDtslLKPQ-SS"\})/\$1\$2/,domain=www.youtube.com
```

Comment:
Rules with `zRgkuMsNLBwOcCE59TDtslLKPQ` don't match on my end
https://regex101.com/r/gKZTp6/1

Reply:
Okay, will change to `[0-9A-Za-z]{9}BwOcCE59TDtslLKPQ`